app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+end
```

Comment:
- [ ] The Comment model should have a method that updates the comments counter for a post.